packages/vertical-layout/README.md

```diff
@@ -0,0 +1,62 @@
+# &lt;vaadin-ordered-layout&gt;
+
+[Live Demo ↗](https://vaadin.com/components/vaadin-ordered-layout/html-examples)
+|
+[API documentation ↗](https://vaadin.com/components/vaadin-ordered-layout/html-api)
+
+[&lt;vaadin-vertical-layout&gt;](https://vaadin.com/components/vaadin-ordered-layout) consist of one of two Web Components providing a simple way to vertically align your HTML elements, part of the [Vaadin components](https://vaadin.com/components).
+
+[![npm version](https://badgen.net/npm/v/@vaadin/veritcal-layout)](https://www.npmjs.com/package/@vaadin/vertical-layout)
+[![Published on Vaadin Directory](https://img.shields.io/badge/Vaadin%20Directory-published-00b4f0.svg)](https://vaadin.com/directory/component/vaadinvertical-layout)
+[![Discord](https://img.shields.io/discord/732335336448852018?label=discord)](https://discord.gg/PHmkCKC)
+
+```html
+<vaadin-vertical-layout>
+  <div>Vertically</div>
+  <div>Aligned</div>
+</vaadin-vertical-layout>
+```
+
+[<img src="https://raw.githubusercontent.com/vaadin/vaadin-ordered-layout/master/screenshot.png" width="200" alt="Screenshot of vaadin-ordered-layout">](https://vaadin.com/components/vaadin-ordered-layout)
```

Comment:
Since `screenshot.png` was removed in this PR, the image will be unavailable as soon as the PR is merged.

Maybe the link to the image should just be removed from `README.md`.

Reply:
The imaged referenced here is one from the `vaadin-ordered-layout` individual repository.

Reply:
Updated README to remove screenshot altogether, we can add them for all new packages later.